packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -115,10 +116,10 @@ const setupDriver = async () => {
 			desiredCaps.app = `sauce-storage:Gutenberg-${ safeBranchName }.apk`; // App should be preloaded to sauce storage, this can also be a URL.
 		}
 	} else {
-		desiredCaps = { ...iosServer };
+		desiredCaps = { ...iosServer( { iPadDevice } ) };
```

Comment:
I may be overlooking something, but the object spread on this line and its sibling seem unnecessary. Feel free to ignore if you disagree.

```suggestion
		desiredCaps = iosServer( { iPadDevice } );
```

Reply:
Nice catch! I updated in https://github.com/WordPress/gutenberg/pull/50528/commits/535548f267b1caeedea276f6400944033526cc87